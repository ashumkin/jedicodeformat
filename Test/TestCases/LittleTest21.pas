unit LittleTest21;

interface

  { this is an enum }
type
    Truncated = (false, true);

implementation

end.
