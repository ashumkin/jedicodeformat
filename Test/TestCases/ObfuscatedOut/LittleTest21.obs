unit littletest21;interface type truncated=(false,true);implementation end. 