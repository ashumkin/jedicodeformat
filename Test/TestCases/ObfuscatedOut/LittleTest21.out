unit littletest21;

interface

type
  truncated = (False, True);

implementation

end.
 